frontend/packages/kubevirt-plugin/src/components/create-vm/dev-console/dev-console-create-vm-form.tsx

```diff
@@ -0,0 +1,241 @@
+                            template,
+                            true,
+                          )} The Boot source is also maintained by Red Hat.`
+                        : `This template is ${getTemplateProvider(t, template, true)}`}
+                    </p>
+                    <>
+                      <hr />
+                      <p>{t('kubevirt-plugin~The following resources will be created')}:</p>
+                      <ul>
+                        <li key="virtual-machine">{t('kubevirt-plugin~Virtual machine')}</li>
+                      </ul>
+                    </>
+                  </div>
+                </div>
+                <div className="col-md-5 col-md-pull-7">
+                  <Stack hasGutter>
+                    <StackItem>
+                      <CreateVMForm
+                        dispatch={dispatch}
+                        state={state}
+                        template={selectedTemplate}
+                        sourceStatus={sourceStatus}
+                        cdRomText={t(
+                          'kubevirt-plugin~A new disk has been added to support the CD-ROM boot source.',
+                        )}
+                        showCreateInfo={false}
+                        showProjectDropdown={false}
+                      />
+                    </StackItem>
+                    <StackItem>
+                      <ButtonBar
+                        inProgress={isSubmitting}
+                        errorMessage={
+                          createError ||
+                          (!isTemplateSourceError(sourceStatus) && sourceStatus?.isCDRom
+                            ? scError
+                            : undefined)
+                        }
+                      >
+                        <ActionGroup className="pf-c-form">
+                          <Button
+                            type="submit"
+                            variant="primary"
+                            isDisabled={
+                              !state.isValid ||
+                              isSubmitting ||
+                              (!isTemplateSourceError(sourceStatus) && sourceStatus?.isCDRom
+                                ? !scLoaded || !!scError
+                                : false)
+                            }
+                            onClick={async () => {
+                              setCreateError(null);
+                              try {
+                                setSubmitting(true);
+                                const vm = await createVM(
+                                  state.template,
+                                  sourceStatus,
+                                  undefined,
+                                  state,
+                                  scConfigMap,
+                                );
+                                history.push(
+                                  `/topology/ns/${getNamespace(vm)}?view=graph&selectId=${getUID(
```

Comment:
To respect the user's preference, do not include the view type. This will bring the user to either the graph or list list view. Unless of course UX wants to force the user to the graph.

```suggestion
                                  `/topology/ns/${getNamespace(vm)}?selectId=${getUID(
```